pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pkgs }:
```

please import libxml directly
```suggestion
{ lib, buildPythonPackage, fetchPypi, libxml2 }:
```